src/components/chart/IntervalTimeline.tsx

```
@@ -0,0 +1,220 @@
+import colors from "@/src/theme/colors";
+import { SegmentInfo, Set } from "@/src/types/pacemaker";
```

Biome (2.1.2)

[error] Do not shadow the global "Set" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)